server/game/cards/11-TCaR/Mausoleum.js

```diff
@@ -0,0 +1,22 @@
+const DeedCard = require('../../deedcard.js');
+
+class Mausoleum extends DeedCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Mausoleum',
+            cost: ability.costs.bootSelf(),
+            when: {
+                onCardAced: event =>
+                    !event.card.hasOneOfKeywords(['token', 'abomination']) && event.card.getType() === 'dude'
```

Comment:
Only triggers for cards aced from play, so needs one more condition:
`event.originalLocation === 'play area'`

Reply:
Added the condition for play area